documentation/docs/examples/cad-prioritized-areas.mdx

```diff
@@ -0,0 +1,45 @@
+---
+id: cad-prioritized-areas
+title: Prioritize Areas for Loading
```

Comment:
```suggestion
title: Prioritize nodes for loading
```

The user with think of this in terms of "nodes" so, that should be our focus too - update filenames etc too